finetuner/tuner/__init__.py

```diff
@@ -46,10 +47,8 @@ def fit(
     :param batch_size: The batch size to use for training and evaluation
     :param loss: Which loss to use in training. Supported
         losses are:
-        - ``CosineSiameseLoss`` for Siamese network with cosine distance
-        - ``EuclideanSiameseLoss`` for Siamese network with eculidean distance
-        - ``CosineTripletLoss`` for Triplet network with cosine distance
-        - ``EuclideanTripletLoss`` for Triplet network with eculidean distance
+        - ``SiameseLoss`` for Siamese network with cosine distance
```

Comment:
losses and distance are not separated?

Reply:
I've refactored this so that they are separated, you can create a loss object with specified distance

Reply:
Correct comment then